tools/docker-builder/types.go

```diff
@@ -94,6 +94,14 @@ func DefaultArgs() Args {
 		"operator",
 		"install-cni",
 
+		"app_sidecar_base_ubuntu_xenial",
```

Comment:
sorry my comment about base wasn't clear - I meant all of the base ones including these.

That will give matching behavior in the old logic and new logic

Reply:
They are removed now. Thanks.